shortfin/python/shortfin_apps/sd/README.md

```diff
@@ -13,6 +13,14 @@ pip install pillow
 pip install shark-ai
 
 ```
+
+Temporarily, you may need an update to your `shortfin` install.
+Choose the latest pre-release from https://github.com/nod-ai/SHARK-Platform/releases/tag/dev-wheels, and install with:
+```
+pip uninstall shortfin -y
+pip install https://<pre-release>.whl
+```
```

Comment:
This should work:
```
pip install shortfin -f https://github.com/nod-ai/SHARK-Platform/releases/expanded_assets/dev-wheels
```

If you pass `--upgrade --pre`, I think it should upgrade in place

developer docs: https://github.com/nod-ai/SHARK-Platform/blob/main/docs/nightly_releases.md

Reply:
works like a charm. Thanks.